blockreassurance.php

```diff
@@ -206,7 +200,7 @@ public function install()
             return true;
         }
 
-        $this->_errors[] = $this->trans('There was an error during the installation. Please contact us through Addons website.', [], 'Modules.Blockreassurance.Admin');
+        $this->_errors[] = $this->trans('There was an error during the installation. Please open an issue on the PrestaShop project https://github.com/PrestaShop/PrestaShop/issues.', [], 'Modules.Blockreassurance.Admin');
```

Comment:
Is there a reason why you left the link visible?
Otherwise, maybe we could write: "There was an error during the installation. Please open an issue on the [PrestaShop project](https://github.com/PrestaShop/PrestaShop/issues).

Reply:
Ping @okom3pom

Reply:
@okom3pom can you follow what @Julievrz suggested, please?